src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Todo = (props) => {
+    return (
+        <div className={props.todo.completed ? "singleTodo completed" : "singleTodo not-completed" } 
```

Comment:
awesome use of the ternary, this is one of the best ways to this on your class. There is also a module called classnames that lets you do something similar but more dynamically based on props.